api/preprints/serializers.py

```diff
@@ -71,6 +71,7 @@ class PreprintSerializer(JSONAPISerializer):
     ])
 
     id = IDField(source='_id', read_only=True)
+    type = TypeField()
```

Comment:
Well, that's a fairly simple solution; very nice. Could you put that field into all of the rest of the serializers?

Reply:
I assume @brianjgeiger meant all JSONAPISerializer based sterilizers, if that makes more tests fail might be worth spinning that as it's own ticket. Not sure.

Reply:
JSONAPISerializer based sterilizers is inherited by 36 Serializer in the code base and a lot of them don't have type field currently and not sure if they necessarily need one, for example, there is a BaseActionSerializer inheriting JSONAPISerializer. This will definitely break a lot of tests and if that's what we want. It's better to make another ticket for this case.